### DISCUSSION TITLE
error when generating pdf on mac

### DISCUSSION THREAD

**Discussion by laravelpro** [honkit/honkit#347] [Q&A]
## Summ
It says: 

  ImportError: dlopen(/Applications/calibre.app/Contents/Frameworks/plugins/podofo.so, 0x0002): Library not loaded: @rpath/libpodofo.1.dylib
    Referenced from: <49CF4C11-44E4-3FBF-8E56-D450E28F5B79> /Applications/calibre.app/Contents/Frameworks/plugins/podofo.so
    Reason: tried: '/usr/lib/libpodofo.1.dylib' (no such file, not in dyld cache)

- HonKit version: latest

Anyone knows how to fix this bug?

**Comment by azu** [comment]
https://github.com/honkit/honkit/tree/master/docker
I recommened to use Docker Image
